### PR TITLE
Use Distroless instead of Alpine

### PR DESCRIPTION
**Pull request by smorimoto**
Ref: https://github.com/GoogleContainerTools/distroless

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,9 @@
 # Production image based on alpine.
```
Comment (lukasmalkmus) [inline Dockerfile:1]: Should adjust the comment, as well.

Comment (smorimoto) [reply Dockerfile]: Good catch!

**Comment by smorimoto** [comment]
Ready to go!

**Review by lukasmalkmus — APPROVED** [review]
LGTM

**Comment by smorimoto** [comment]
When is this going to be released?

**Comment by lukasmalkmus** [comment]
Now :)

**Comment by smorimoto** [comment]
Fantastic! Thank you! 🙂

**Pull request merged**